cnd/src/swap_protocols/hbit/ledger_states.rs

```diff
@@ -0,0 +1,84 @@
+use crate::swap_protocols::{
+    hbit::{LedgerState, SwapEvent},
+    LocalSwapId,
+};
+use async_trait::async_trait;
+use std::collections::HashMap;
+use tokio::sync::Mutex;
+
+#[async_trait]
+pub trait Insert: Send + Sync + 'static {
```

Comment:
❓ I could just be very far behind in terms of this codebase, but why are we not using the traits provided by `swap_protocols/state.rs`?

Reply:
Good and fair point, I will investigate.  Discussed already with @da-kami and @thomaseizinger and will either
- Use traits in `swap_protocols/state.rs`
- Remove the traits and just have methods.

Reply:
Removed the traits. Thanks.